### PR TITLE
Mock API calls for the ticket audit tests

### PR DESCRIPTION
**Pull request by miogalang**
Mock the API calls for ticket audit test. Removed unnecessary tests

/cc @joseconsador @jwswj @mmolina @atroche 

### References
 - Jira link: 

### Risks
 - Tests might fail (low)

### CODE REVIEW

**Comment by joseconsador** [comment]
:+1:

File: tests/Zendesk/API/LiveTests/TicketAuditsTest.php [hunk 1]
```diff
@@ -2,69 +2,82 @@
...
             ),
             'priority' => 'normal'
         );
-        $ticket = $this->client->tickets()->create($testTicket);
-
-        $update_testTicket['id'] = $ticket->ticket->id;
-        $update_testTicket2['subject'] = 'Updated subject';
-        $update_testTicket2['priority'] = 'urgent';
-        $ticket = $this->client->tickets()->update($update_testTicket);
+//        $ticket = $this->client->tickets()->create($testTicket);
```
Comment (atroche) [inline tests/Zendesk/API/LiveTests/TicketAuditsTest.php]: did you mean to leave this in?

**Comment by atroche** [comment]
:+1:

File: tests/Zendesk/API/LiveTests/TicketAuditsTest.php (hunk 1 above)
Comment (miogalang) [inline tests/Zendesk/API/LiveTests/TicketAuditsTest.php]: Great catch. Missed this

**Pull request merged**